### PR TITLE
added field name and column for optional label (Issue: Field creation #31)

### PR DESCRIPTION
**Pull request by tobischulz**
This will add a optional column label for each field.

## Usage:

```php
<?php
public function fields(): array
{
    return [
        Input::make('name', 'The Username'),
    ];
}
```

First parameter will be the name of the field that is the database column name. 
Second is the optional Label for the UI. Guess the naming of the variables inside the NebulaField.php is missleading here. :/

### CODE REVIEW

**Comment by tobischulz** [comment]
Renamed the variable inside the NebulaField.php to "$label" that should make much more sense now.

File: src/Contracts/NebulaField.php [hunk 1]
```diff
@@ -17,21 +17,24 @@ abstract class NebulaField
      * Construct the field.
      *
      * @param string $name
+     * @param string $label
      */
-    public function __construct(string $name)
+    public function __construct(string $name, string $label = null)
```
Comment (Larsklopstra) [inline src/Contracts/NebulaField.php]: Could you reverse the order and generate a name based on the label? ex:

```
Input::make('Name')
> label: Name
> name: name

Input::make('Order Status')
> label: Order Status
> name: order_status
```

**Review by Larsklopstra — CHANGES_REQUESTED** [review]

File: src/Contracts/NebulaField.php (hunk 1 above)
Comment (tobischulz) [reply src/Contracts/NebulaField.php]: Sure, i'm on it.

Comment (tobischulz) [reply src/Contracts/NebulaField.php]: done 😄

**Pull request merged**